test/e2e/cli-install-uninstall/01-upgrade.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kudo.dev/v1alpha1
```

Comment:
what's the point of testing upgrade in uninstall test? I think tests should be focused and not duplicated much and we already have upgrade test

Reply:
in this case it's not even upgrade but just update of an parameter? 🤔 in any case I think there should be a focused test on that (we already have one I believe) and this one should just install and uninstall

Reply:
You're right, we don't need to upgrade here. That's because I originally intended to add uninstall to `integration/cli-install`. Now that this is a separate E2E test, the upgrade can be removed.